CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 
 ## [Unreleased]
+- Add a file picker dialog to choose where to export a CSV to
```

Comment:
shouldn't this be in 4.31.0 below?

Reply:
No, as it is not in the release. RC1 is already tagged and built, adding features after that is not a good idea.

Reply:
oh, i thought filepicker was going in the 4.31.